tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -21,11 +21,11 @@ sub run {
     # when some module (e.g. workstation extension) requires registration, provide separate code
     my $timeout = 60 * get_var('TIMEOUT_SCALE', 1);
     my $regcode = get_var('SCC_REGCODE_WE');
-    $testapi::distri->get_module_regcode()->add_separate_registration_code($regcode, $timeout);
+    $testapi::distri->get_module_regcode()->add_separate_registration_code($regcode, $timeout) if (check_var('ARCH', 'x86_64'));
```

Comment:
again, there are no conditions in test modules, just ignore the other architectures.

Reply:
Have removed. I added this condition to do VR. 
I have finished VR. Removed the code